diff --git a/plugin.php b/plugin.php
index 2ec0e3d..166b2e1 100644
--- a/plugin.php
+++ b/plugin.php
@@ -26,6 +26,8 @@ while($plugin=$qry2->fetch())
     if($section=='main' && file_exists('plugins/'.$plugin['name'].'/main.php')) {include('plugins/'.$plugin['name'].'/main.php');}
     //page white list
     if($section=='page_white_list' && file_exists('plugins/'.$plugin['name'].'/page_white_list.php')) {include('plugins/'.$plugin['name'].'/page_white_list.php');} 
+    //azure ad auth success (successful Entra ID SSO login)
+    if($section=='azure_ad_auth_success' && file_exists('plugins/'.$plugin['name'].'/azure_ad_auth_success.php')) {include('plugins/'.$plugin['name'].'/azure_ad_auth_success.php');}
     //favicon
     if($section=='favicon' && file_exists('plugins/'.$plugin['name'].'/favicon.php')) {include('plugins/'.$plugin['name'].'/favicon.php');} 
     //breadcrumb
